share/shwap/p2p/shrex/shrex_getter/shrex_test.go

```diff
@@ -1,4 +1,4 @@
-package getters
+package shrex_getter
```

Comment:
why subpkg?

Reply:
We can always change that later ofc, but for now wanna know if there was a cycle or smth

Reply:
cycle